nixos/modules/services/monitoring/prometheus/exporters/rtl_433.nix

```diff
@@ -0,0 +1,84 @@
+{ config, lib, pkgs, options }:
```

Comment:
Why is "options" in here? AFAICT it's not used anywhere.

Reply:
It's not used but it's [passed by Prometheus import](https://github.com/NixOS/nixpkgs/blob/97005a006c7a62aebdd79ccc2b2eb179991465bd/nixos/modules/services/monitoring/prometheus/exporters.nix#L53). Removing it fails the build.